HIP/hip-0000-cross-ledger-protocol-clpr.md

```diff
@@ -0,0 +1,1152 @@
+---
+hip: <HIP number (assigned by the HIP editor), usually the PR number>
+title: Cross-Ledger Protocol (CLPR)
+author: Edward Wertz <@edward-swirldslabs>
+working-group: Leemon Baird <leemon@hashgraph.com>, Richard Bair <@rbair32>, Jasper Potts <@jasperpotts>, Joseph Sinclair <@jsync-swirlds>, Rohit Sinha <@rsinha>, Kiran Pachhai <@kpachhai>, Torfin Olsen <torfinn@hashgraph.com>, Edward Wertz <@edward-swirldslabs>
+requested-by: Leemon Baird <leemon@hashgraph.com>
+type: "Standards Track"
+category: "Core"
+needs-council-approval: "Yes"
+status: "Idea"
+created: 2026-06-18
+discussions-to:
+  <A URL pointing to the official discussion thread. Ex: https://github.com/hashgraph/hedera-improvement-proposal/discussions/000>
+updated: <Latest date HIP was updated, in YYYY-MM-DD format.>
+requires: 1200
+replaces: N/A
+superseded-by: N/A
+---
+
+## Abstract
+
+This HIP introduces the `Cross-Ledger PRotocol` or `CLPR`, pronounced `Clipper`
+like the type of ship. `CLPR` or `Clipper` is an interledger communication
+protocol that uses a connector abstraction to manage the logistics of payments
+and sending application messages between different ledgers. The use of the
+connector abstraction allows for a clean separation of concerns between
+application logic and the mechanics of sending messages via state proofs of
+connector state. Combining the use of state proofs to establish message
+order with the semantics of guaranteed, exactly once, in-order message
+handling, Clipper is able to preserve the asynchronous Byzantine
+Fault Tolerant (aBFT) security profile of an already aBFT secure ledger.
+
+![Separable Connector and Application](../assets/hip-0000-clpr/clpr-simple-high-level-diagram.drawio.svg)
+
+Two Clipper enabled ledgers can communicate directy with each other without
+the use of intermediate relays, bridges, or oracles. If both ledgers are
+aBFT, then the combined interaction is aBFT. If one ledger was
+only BFT while the other was aBFT, the combined interaction is BFT. Clipper
+preserves the weaker security profile of the interacting ledgers.
+
+Clipper is extensible in two ways. The `Clipper Application Protocol`
```

Comment:
I'm not sure if this paragraph is useful. I am still trying to wrap my head around it after 3-4 readings.